packages/hds/src/components/Gauge/Gauge.tsx

```diff
@@ -0,0 +1,34 @@
+import { type HTMLAttributes, forwardRef } from 'react';
+import * as styles from './Gauge.css';
+
+export interface GaugeProps extends HTMLAttributes<HTMLDivElement> {
+  /** 게이지 바 연한 컬러 */
+  bgColor?: string;
+  /** 게이지 컬러 */
+  gauageColor?: string;
+  /** 게이지에 표시할 퍼센테지 @default 0 */
+  progress?: number;
+}
+
+const Gauge = forwardRef<HTMLDivElement, GaugeProps>(
+  ({ bgColor, gauageColor, progress = 0, ...rest }, ref) => {
+    return (
+      <div
+        className={styles.guageBackground}
```

Comment:
className clsx 사용해서 props 의 className과 병합해주세요.
```suggestion
        className={clsx(styles.guageBackground, className)}
```

Reply:
> className clsx 사용해서 props 의 className과 병합해주세요.

요렇게 사용해야 하는 이유가 있을까요? 조건부 스타일링이나, className을 여러개 사용하는 상황이 아니라서 위와 같은 상황에서는 변경 전후가 차이가 없어보여서요~

Reply:
사용하는 쪽에서 추가적으로 스타일링을 가능하게 하기 위해서 className 을 기존의 className 과 병합하는 겁니다.

**예시**
- MUI 같은 UI 라이브러리를 사용할 때에 className, sx, style 등을 넘겨서 추가적으로 스타일링을 해보신 경험이 있으실 거란 생각이 드는데요, 그럴 때를 위해 저렇게 작성하는 겁니다.

@say-young516